lib/Service/AccountService.php

```diff
@@ -43,12 +45,17 @@ class AccountService {
 	/** @var IL10N */
 	private $l10n;
 
+	/** @var Manager */
+	private $defaultAccountManager;
+
 	/**
 	 * @param MailAccountMapper $mapper
 	 */
-	public function __construct(MailAccountMapper $mapper, IL10N $l10n) {
+	public function __construct(MailAccountMapper $mapper, IL10N $l10n,
+		Manager $defaultAccountManager) {
```

Comment:
My bad, of course they should be in sync with the actual parameters of the method :-)